core-common/src/main/java/org/zalando/nakadi/repository/zookeeper/ZooKeeperHolder.java

```diff
@@ -112,26 +107,7 @@ private CuratorFramework createCuratorFramework(final int sessionTimeoutMs,
     private EnsembleProvider createEnsembleProvider() throws Exception {
         switch (conn.getType()) {
             case EXHIBITOR:
-                final Exhibitors exhibitors = new Exhibitors(
-                        conn.getAddresses().stream().map(AddressPort::getAddress).collect(Collectors.toList()),
-                        conn.getAddresses().get(0).getPort(),
-                        () -> {
-                            throw new RuntimeException("There is no backup connection string (or it is wrong)");
-                        });
-                final ExhibitorRestClient exhibitorRestClient = new DefaultExhibitorRestClient();
-                final ExhibitorEnsembleProvider result = new ExhibitorEnsembleProvider(
-                        exhibitors,
-                        exhibitorRestClient,
-                        "/exhibitor/v1/cluster/list",
-                        EXHIBITOR_POLLING_MS,
-                        new ExponentialBackoffRetry(EXHIBITOR_RETRY_TIME, EXHIBITOR_RETRY_MAX)) {
-                    @Override
-                    public String getConnectionString() {
-                        return super.getConnectionString() + conn.getPathPrepared();
-                    }
-                };
-                result.pollForInitialEnsemble();
-                return result;
+                throw new RuntimeException("Exhibitor support is removed on 2020-12-15; use zookeeper connection");
```

Comment:
I'm a bit concerned about this. In order not to flood - let's just remove this enum constant at all?